app/domain/errors.rb

```diff
@@ -484,6 +484,22 @@ module AuthnJwt
         code: "CONJ00102E"
       )
 
+      FailedToValidateClaimMissingClaimName = ::Util::TrackableErrorClass.new(
+        msg: "Failed to validate claim, claim name is empty, or was not found.",
+        code: "CONJ00103E"
+      )
+
+      FailedToValidateClaimForbiddenClaimName = ::Util::TrackableErrorClass.new(
+        msg: "Failed to validate claim, claim name '{0-claim-name}' is forbidden, " \
+             " please validate all claims defined in '{1-mandatory-claims-variable-name}' " \
+             "are matching '{2-regex}' regular expression.",
+        code: "CONJ00104E"
+      )
+
+      FailedToValidateClaimClaimNameInDenyList  = ::Util::TrackableErrorClass.new(
+        msg: "Failed to validate claim, claim name '{0-claim-name}' is in deny list '{1-deny-list}'",
+        code: "CONJ00105E"
+      )
```

Comment:
1. I'm not sure we the phrase 'claim name' is correct, I think it should be either claim or claim key.
2. denylist in one word -> see [this](https://ca-il-confluence.il.cyber-ark.com/display/rndp/%5BSpike%5D+Rename+and+offensive+of+conjur#id-[Spike]Renameandoffensiveofconjur-Specificchanges)

Reply:
@shulifink WDYT?

Reply:
@hilagross "claim name" is looking like appropriate term see [RFC](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1).

Reply:
great, so we should keep it :)
still need to change denylist

Reply:
Done